pyvirtualcam/camera.py

```diff
@@ -4,6 +4,7 @@
 from abc import ABC, abstractmethod
 
 import numpy as np
+from numpy import asarray
```

Comment:
Can you revert all changes to this file?

Reply:
Yes, 
sorry about the mistakes, I'm new to contributing to open source.

Reply:
No worries, all good :)